kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+
+  <div>
+    <KGrid
+      class="add-padding"
+    >
+      <KGridItem
+        :layout4="{ span: 1 }"
+        :layout8="{ span: 1 }"
+        :layout12="{ span: 1 }"
+      >
+        <KIcon
+          icon="quiz"
+          class="style-icon"
+        />
+      </KGridItem>
+
+      <KGridItem
+        :layout4="{ span: 3 }"
+        :layout8="{ span: 7 }"
+        :layout12="{ span: 11 }"
+      >
+        <KTextbox
+          ref="title"
+          v-model.trim="examTitle"
+          :label="coachString('titleLabel')"
+          :autofocus="true"
+          :maxlength="100"
+        />
+      </KGridItem>
+    </KGrid>
+
+    <p>{{ $tr('addSectionsDescription') }}</p>
+
+    <hr class="bottom-border">
+    <br>
+
+    <KGrid style="margin:0;padding:0">
+      <KGridItem
+        :layout12="{ span: 8 }"
+      >
+        <span class="active-section">
+          {{ ($tr('sectionLabel')).toUpperCase() }}
+        </span>
+      </KGridItem>
+
+      <KGridItem
+        :layout12="{ span: 4 }"
+      >
+        <KButton
+          class="add-section-button"
+          appearance="flat-button"
+          icon="plus"
+        >
+          {{ ($tr('addSection')).toUpperCase() }}
+        </KButton>
+      </KGridItem>
+    </KGrid>
+
+    <hr class="bottom-border">
```

Comment:
I'm not sure if you'll need to do a bit of CSS tweaking but this isn't sitting flushly against the horizontal line break.

Expected:

![image](https://github.com/learningequality/kolibri/assets/6356129/ece6f800-0adb-42b4-a828-5d9f351fb254)

In browser:

![image](https://github.com/learningequality/kolibri/assets/6356129/450aaa57-d58a-4cad-8b0a-6f32d7b6489b)

Reply:
Resolved in last updated